libraries/disp/viewers/scalingview.h

```diff
@@ -173,6 +192,19 @@ class DISPSHARED_EXPORT ScalingView : public AbstractView
 
     Ui::ScalingViewWidget*              m_pUi;
 
+    //=========================================================================================================
+    /**
+     * Default scales for each channel by type
+     */
+    static float                         d_ScaleMAG;
```

Comment:
Pleae document and rename to `m_dScaleMAG`. Same goes for the other default values. Maybe we should also consider using `const static` so that the defaults can not be changed during runtime. What do you think?